build.sbt

```diff
@@ -18,13 +13,14 @@ import DefaultBuildSettings._
 import uk.gov.hmrc.SbtAutoBuildPlugin
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
 import uk.gov.hmrc.versioning.SbtGitVersioning
-import uk.gov.hmrc.SbtArtifactory
 
 lazy val plugins : Seq[Plugins] = Seq.empty
 lazy val playSettings : Seq[Setting[_]] = Seq.empty
 
+lazy val appName = "valuation-office-agency-contact-frontend"
+
 lazy val root = Project(appName, file("."))
-  .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory) ++ plugins : _*)
```

Comment:
Did you also removed `sbt-artifactory` plugin?

Reply:
The sbt-artifactory had already been removed from the plugin.sbt file by your good self 4 months ago.